tactis/model/decoder.py

```diff
@@ -274,76 +349,89 @@ def __init__(
         self.mlp_dim = mlp_dim
         self.resolution = resolution
         self.dropout = dropout
-        self.fixed_permutation = fixed_permutation
 
+        output_dims = self.attention_heads * self.attention_dim
         # Parameters for the attention layers in the copula
         # For each layer and each head, we have two MLP to create the keys and values
         # After each layer, we transform the embedding using a feed-forward network, consisting of
         # two linear layer with a ReLu in-between both
         # At the very beginning, we have a linear layer to change the embedding to the proper dimensionality
         self.dimension_shifting_layer = nn.Linear(self.input_dim, self.attention_heads * self.attention_dim)
 
+        if activation_function == "relu":
+            activation = nn.ReLU
```

Comment:
Missing a else statement to throw an error.

Reply:
Resolved.